### DISCUSSION TITLE
Stop parallax layer to scroll

### DISCUSSION THREAD

**Discussion by simerdev** [pmndrs/react-spring#885] [Support]
## 🤓 Question

I need a parallax layer to scroll with the page scroll and then stop it at a point, so that it looked as a fixed, even when the page is keep on scrolling. 

Need urgent help on this.

**Comment by kindoflew** [comment]
Sorry this has sat here for so long! This is now possible with the new `sticky` prop which was added in v9.2.0.

**Reply by jibrantannous** [reply]
Hi! I have a problem with the sticky prop. I'm trying to set a sticky parallax Layer, which is the father of an image. I am positioning that image at the bottom of the page/layer as an initial position of the sticky prop, then I want that image to be in the same position until the end of the 5 page. But when I scrolled, the image is position in the middle, is not sticky in the same point along the whole parallax container. Do you know how to set that behavior?
